pwiz_tools/Skyline/Test/CodeInspectionTest.cs

```diff
@@ -215,6 +219,50 @@ List<string> CheckFormsWithoutTestRunnerLookups()
             return missing;
         }
 
+        /// <summary>
+        /// We have some code, especially in commandline, that expects certain error messages to start with "Error:" or the L10N equivalent
+        /// </summary>
+        void InspectConsistentErrorMessages(List<string> errors)
+        {
+            var currentCulture = Thread.CurrentThread.CurrentUICulture;
+            try
+            {
+                Thread.CurrentThread.CurrentCulture = Thread.CurrentThread.CurrentUICulture = new CultureInfo("en"); // We want to compare against the "en" resources
+                var resourceSet = Skyline.Properties.Resources.ResourceManager.GetResourceSet(Thread.CurrentThread.CurrentUICulture, true, true);
+
+                // Build the string of resources.
+                foreach (var resource in resourceSet)
+                {
+                    var pair = resource as DictionaryEntry? ?? default;
+                    var val = pair.Value as string ?? string.Empty;
+                    if (val.StartsWith("Error:"))
+                    {
+                        var name = pair.Key.ToString();
+                        foreach (var culture in new[] {@"zh-CHS", @"ja"}) 
+                        {
+                            var tryCulture = new CultureInfo(culture);
+                            Thread.CurrentThread.CurrentCulture = Thread.CurrentThread.CurrentUICulture = tryCulture;
+                            var local = Skyline.Properties.Resources.ResourceManager.GetString(name) ?? String.Empty;
+                            var commandStatusWriterWriteLineError = Skyline.Properties.Resources.CommandStatusWriter_WriteLine_Error_;
+                            if (!local.StartsWith(commandStatusWriterWriteLineError, StringComparison.CurrentCulture) &&
+                                !local.StartsWith(@"Error:", StringComparison.InvariantCulture)) // Maybe not yet localized
```

Comment:
Way too many uses of "Error:" typed into text.

Reply:
Yup.